packages/react-docs/src/utils/typedoc_utils.ts

```diff
@@ -85,6 +85,16 @@ export const typeDocUtils = {
             let entities;
             let packageComment = '';
             // HACK: We assume 1 exported class per file
+            const numClassChildren = _.reduce(
```

Comment:
Let's use `_.filter`. Using reduce in this context is redundant. Also - reduce should have an explicit accumulator for readability purposes. (In future).